prov/efa/configure.m4

```diff
@@ -31,9 +40,58 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	      [AC_MSG_RESULT([no])
 	       efa_happy=0])
 
-	# verbs definitions file depends on linux/types.h
-	AC_CHECK_HEADER([linux/types.h], [], [efa_happy=0])
-
+	AS_IF([test x"$enable_efa" != x"no"],
+	      [FI_CHECK_PACKAGE([efadv],
+				[infiniband/efadv.h],
+				[efa],
+				[efadv_query_ah],
+				[-libverbs],
+				[$efa_PREFIX],
+				[$efa_LIBDIR],
+				[efa_happy=1],
+				[efa_happy=0])
+	      ])
```

Comment:
Would you please add a check for function efadv_query_device() too? That function was also used by this patch.

Reply:
Sure, I've added a check for that.